bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/wrappers/veneer/TestBigtableHBaseVeneerSettings.java

```diff
@@ -358,17 +334,10 @@ public void testAdminSettingsWithEmulator() throws IOException {
   @Test
   public void testRpcMethodWithoutRetry() throws IOException {
     configuration.set(ENABLE_GRPC_RETRIES_KEY, "false");
-    BigtableHBaseVeneerSettings settingUtils =
-        ((BigtableHBaseVeneerSettings) BigtableHBaseVeneerSettings.create(configuration));
+    BigtableHBaseVeneerSettings settingUtils = BigtableHBaseVeneerSettings.create(configuration);
     BigtableDataSettings settings = settingUtils.getDataSettings();
 
     assertTrue(settings.getStubSettings().readRowsSettings().getRetryableCodes().isEmpty());
-
-    assertRpcWithoutTimeout(settings.getStubSettings().bulkReadRowsSettings());
```

Comment:
can we add new tests in its place?

Reply:
Not sure what you mean? This used to test that timeouts were forced to 12 hours. Are you saying that I should add a test that disabling retries doesn't affect timeouts?

Reply:
yes that would be good